### PR TITLE
Docs: Cleanup metadata and improve consistency

### PR DESCRIPTION
**Pull request by josmperez**
Cleanup metadata and improve consistency.

### CODE REVIEW

File: docusaurus/docs/get-started/folder-structure.md [hunk 1]
```diff
@@ -1,6 +1,13 @@
 ---
 id: folder-structure
 title: Folder structure
+description: How your folder structure should look after installing create-plugin.
```
Comment (sympatheticmoose) [inline docusaurus/docs/get-started/folder-structure.md:4]:
```suggestion
description: How your folder structure should look after running create-plugin.
```

File: docusaurus/docs/get-started/get-started.mdx [hunk 2]
```diff
@@ -1,6 +1,13 @@
 ---
 id: get-started
 title: Get started
+description: Get started with Grafana plugin developing with the create-plugin tool.
```
Comment (sympatheticmoose) [inline docusaurus/docs/get-started/get-started.mdx:4]:
```suggestion
description: Get started with Grafana plugin development with the create-plugin tool.
```

File: docusaurus/docs/introduction/backend.md [hunk 3]
```diff
@@ -1,6 +1,13 @@
 ---
 id: backend-plugins
 title: Backend plugins
+description: Learn about the Grafana plugin system for extending the features of Grafana.
```
Comment (sympatheticmoose) [inline docusaurus/docs/introduction/backend.md:4]:
```suggestion
description: Learn about the Grafana backend plugin system for extending the features of Grafana.
```

File: docusaurus/docs/migration-guides/update-from-grafana-versions/v6.x-v7.x.md [hunk 4]
```diff
@@ -2,7 +2,14 @@
 id: migrate-6.x-to-7.0
 title: 6.x to 7.0
 sidebar_position: 8
-description: "Instructions for upgrading Grafana from version 6.x to 7.0."
+description: How to upgrade plugins from Grafana  version 6.x to 7.0.
```
Comment (sympatheticmoose) [inline docusaurus/docs/migration-guides/update-from-grafana-versions/v6.x-v7.x.md:5]:
```suggestion
description: How to upgrade plugins from Grafana version 6.x to 7.0.
```

File: docusaurus/docs/migration-guides/update-from-grafana-versions/v7.x-v8.x.md [hunk 5]
```diff
@@ -2,7 +2,14 @@
 id: migrate-7.x.x-to-8.x.x
 title: 7.x.x to 8.x.x
 sidebar_position: 7
-description: "Follow these instructions to migrate Grafana v7.x.x plugins to the updated plugin system available in Grafana v8.x.x."
+description: How to migrate Grafana v7.x.x plugins to the updated plugin system available in Grafana v8.x.x.
```
Comment (sympatheticmoose) [inline docusaurus/docs/migration-guides/update-from-grafana-versions/v7.x-v8.x.md:5]: Maybe just 7.x to 8.x? Same for title. Thinking wrt consistency with v6 docs?

File: docusaurus/docs/migration-guides/update-from-grafana-versions/v8.3.x-8.4.x.md [hunk 6]
```diff
@@ -2,7 +2,14 @@
 id: migrate-8.3.x-to-8.4.x
 title: 8.3.x to 8.4.x
 sidebar_position: 6
-description: "Follow these instructions to migrate Grafana v8.3.x plugins to the updated plugin system available in Grafana v8.4.x."
+description: How to migrate Grafana v8.3.x plugins to the updated plugin system available in Grafana v8.4.x.
```
Comment (sympatheticmoose) [inline docusaurus/docs/migration-guides/update-from-grafana-versions/v8.3.x-8.4.x.md:5]: I'm not sure about "plugin system", but also it doesn't seem to be consistently applied in this section

**Review by sympatheticmoose — APPROVED** [review]
Some minor suggestions, otherwise LGTM

File: docusaurus/docs/migration-guides/update-from-grafana-versions/v7.x-v8.x.md (hunk 5 above)
Comment (josmperez) [reply docusaurus/docs/migration-guides/update-from-grafana-versions/v7.x-v8.x.md:5]: Fixed.

File: docusaurus/docs/migration-guides/update-from-grafana-versions/v8.3.x-8.4.x.md (hunk 6 above)
Comment (josmperez) [reply docusaurus/docs/migration-guides/update-from-grafana-versions/v8.3.x-8.4.x.md:5]: Fixed.

File: docusaurus/docs/migration-guides/update-from-grafana-versions/v8.3.x-8.4.x.md [hunk 7]
```diff
@@ -2,12 +2,19 @@
 id: migrate-8.3.x-to-8.4.x
 title: 8.3.x to 8.4.x
 sidebar_position: 6
-description: "Follow these instructions to migrate Grafana v8.3.x plugins to the updated plugin system available in Grafana v8.4.x."
+description: How to migrate Grafana v8.3.x plugins to  Grafana v8.4.x.
```
Comment (sympatheticmoose) [inline docusaurus/docs/migration-guides/update-from-grafana-versions/v8.3.x-8.4.x.md:5]:
```suggestion
description: How to migrate Grafana v8.3.x plugins to Grafana v8.4.x.
```

**Review by sympatheticmoose — APPROVED** [review]
Just one more missed space

**Comment by sympatheticmoose** [comment]
Merging since Joe is out

**Pull request merged**